ProcessLib/ComponentTransport/Tests.cmake

```diff
@@ -898,6 +898,72 @@ AddTest(
     theis_pcs_0_ts_73_t_100000.000000.vtu darcy_velocity darcy_velocity 1e-10 1e-6
 )
 
+AddTest(
+    NAME ComponentTransport_ConTracer_1d
+    PATH Parabolic/ComponentTransport/ConTracer
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS ConTracer_1d.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
+    RUNTIME 10
+    DIFF_DATA
+    ConTracer_1d_pcs_1_ts_33_t_118800.000000_expected.vtu ConTracer_1d_pcs_1_ts_33_t_118800.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_66_t_237600.000000_expected.vtu ConTracer_1d_pcs_1_ts_66_t_237600.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_99_t_356400.000000_expected.vtu ConTracer_1d_pcs_1_ts_99_t_356400.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_132_t_475200.000000_expected.vtu ConTracer_1d_pcs_1_ts_132_t_475200.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_165_t_594000.000000_expected.vtu ConTracer_1d_pcs_1_ts_165_t_594000.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_198_t_712800.000000_expected.vtu ConTracer_1d_pcs_1_ts_198_t_712800.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_231_t_831600.000000_expected.vtu ConTracer_1d_pcs_1_ts_231_t_831600.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_264_t_950400.000000_expected.vtu ConTracer_1d_pcs_1_ts_264_t_950400.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_297_t_1069200.000000_expected.vtu ConTracer_1d_pcs_1_ts_297_t_1069200.000000.vtu pressure pressure 1e-6 1e-10
+    ConTracer_1d_pcs_1_ts_329_t_1184400.000000_expected.vtu ConTracer_1d_pcs_1_ts_329_t_1184400.000000.vtu pressure pressure 1e-6 1e-10
```

Comment:
Maybe you only need to compare the results of the final step, say ConTracer_1d_pcs_1_ts_329_t_1184400.000000.vtu.